CONTRIBUTING_DOCS.md

```diff
@@ -1,18 +1,29 @@
 # Contribute to Documentation
 
 If you want to contribute to a project documentation and make it better, your help is very welcome.
-This guide puts together the guidelines to help you figure out how you can offer your feedback and contribute to the documentation.
+This guide puts together the guidelines to help you figure out how you can offer your feedback and contribute to the OpenVINO™ toolkit documentation.
```

Comment:
This reads a bit confusing and guide plus guidelines in the same sentence is redundant. May something shorter like this: “Below are guidelines for offering feedback and contributions to the OpenVINO toolkit documentation.”